server/src/services/session.service.spec.ts

```diff
@@ -26,6 +27,14 @@ describe('SessionService', () => {
     expect(sut).toBeDefined();
   });
 
+  describe('search', () => {
```

Comment:
Should we test deleting outdated tokens and preserving recent tokens?

Reply:
Yes, had to run though and that's all I finished on the first pass lol

Reply:
All good. Should this PR still be draft then? 😛

Reply:
Nope I'm on it :)

Reply:
Ideally I'd also like to have e2e tests for testing the repository method but since this is a nightly job that's a bit tricky I think.